### PR TITLE
Move Tenant from Modal to Page

### PR DESCRIPTION
**Pull request by dvaldivia**
Screen Shot 2021-02-10 at 1 36 04 PM

### CODE REVIEW

**Review by bexsoft — APPROVED** [review]

File: pkg/acl/endpoints.go [hunk 1]
```diff
@@ -32,6 +32,7 @@ var (
 	bucketsDetail       = "/buckets/:bucketName"
 	serviceAccounts     = "/account"
 	tenants             = "/tenants"
+	addTenant           = "/add-tenant"
```
Comment (bexsoft) [inline pkg/acl/endpoints.go:35]: Please update endpoints list in tests

File: portal-ui/src/screens/Console/Common/GenericWizard/GenericWizard.tsx [hunk 2]
```diff
@@ -51,8 +56,21 @@ const styles = (theme: Theme) =>
         boxShadow: "none",
       },
     },
+    paddedGridItem: {
+      padding: "0px 10px 0px 10px",
+    },
+    menuPaper: {
+      // margin: "10px 0px 10px 0px",
+      // padding: "10px 0px 10px 0px",
+      // maxWidth: "320px",
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Common/GenericWizard/GenericWizard.tsx:65]: Please remove these lines

File: portal-ui/src/screens/Console/Common/GenericWizard/GenericWizard.tsx [hunk 3]
```diff
@@ -51,8 +56,21 @@ const styles = (theme: Theme) =>
...
+    },
+    paperContainer: {
+      padding: "10px",
+      maxWidth: "900px",
+    },
     wizardContainer: {
-      flexGrow: 1,
+      // padding: "10px 0px 10px 0px",
     },
   });
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Common/GenericWizard/GenericWizard.tsx:75]: Please remove this class if not used anymore

File: portal-ui/src/screens/Console/Tenants/ListTenants/AddTenant.tsx [hunk 4]
```diff
@@ -1179,7 +1190,8 @@ const AddTenant = ({
           };
 
           setAddSending(false);
-          closeModalAndRefresh(true, newSrvAcc);
+          // closeModalAndRefresh(true, newSrvAcc);
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Tenants/ListTenants/AddTenant.tsx:1193]: Please remove this

**Review by bexsoft — CHANGES_REQUESTED** [review]

File: pkg/acl/endpoints.go (hunk 1 above)
Comment (dvaldivia) [reply pkg/acl/endpoints.go:35]: done

**Review by Alevsk — APPROVED** [review]

**Review by bexsoft — APPROVED** [review]

**Pull request merged**